client/app/views/MenuItemView.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, TouchableOpacity, Text, ScrollView } from 'react-native';
+import { Transition } from 'react-navigation-fluid-transitions';
+
+import connectToRedux from '../redux/lib/connectToRedux';
+import sp from '../redux/lib/stateProperties';
+
+import Header from '../components/Header';
+import MenuItem from '../components/MenuItem';
+import BottomTabs from '../components/BottomTabs';
+
+import styles from '../config/styles';
+
+class MenuItemView extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidMount() {
+        // TODO: make sure that this is the appropriate interface to get the itemID
+        this.props.getMenuItemInformation(this.props.menu.data.itemID);
```

Comment:
Yep! Sounds good. In fact, this can be done independent of the menu list view, so what you did is solid. ✔️

Reply:
Not quite sure how to pull from the store...do I need to create another action?